web/app.js

```diff
@@ -1908,8 +1970,8 @@ window.addEventListener('keydown', function keydown(evt) {
         break;
       case 71: // g
         if (!PDFViewerApplication.supportsIntegratedFind) {
-          PDFViewerApplication.findBar.dispatchEvent('again',
-                                                     cmd === 5 || cmd === 12);
+          PDFViewerApplication.eventBus.dispatch('findagain',
+                                                 cmd === 5 || cmd === 12);
```

Comment:
This doesn't look right, and it stops <kbd>Ctrl</kbd>+<kbd>G</kbd> from working in the preview.